src/ui/screen_process.c

```diff
@@ -34,7 +34,7 @@ void ui_screen_render_component(component_t* component)
     UG_SendBuffer();
 }
 
-static component_t *get_waiting_screen(void)
+static component_t* _get_waiting_screen(void)
```

Comment:
avoid merging fixup commits, rather squash them into the right commit, to keep a nice commit history

`git blame` should show relevant commits mostly, not "fixed this and that" :)

Reply:
Got it :)